### PR TITLE
Update archer quest PRE-RE

### PR DESCRIPTION
**Pull request by elanosilveira**
Missing else statement.

* **Addressed Issue(s)**: 
Small fix for an old NPC. Missing an ELSE statement. The player needed to interact with the NPC twice for no reason.

* **Server Mode**: 
PRE-RE

* **Description of Pull Request**:

### CODE REVIEW

File: npc/pre-re/jobs/1-1/archer.txt [hunk 1]
```diff
@@ -117,10 +117,12 @@ payon_in02,64,71,4	script	Archer Guildsman#archer	85,{
 					close;
 				}
 			}
+			else {
 			mes "[Archer Guildsman]";
 			mes "Well, alright.";
 			mes "See you next time.";
 			close;
```
Comment (Daegaladh) [inline npc/pre-re/jobs/1-1/archer.txt:124]:
```suggestion
				mes "[Archer Guildsman]";
				mes "Well, alright.";
				mes "See you next time.";
				close;
```

**Review by Daegaladh — CHANGES_REQUESTED** [review]
Great, just add the indentation

File: npc/pre-re/jobs/1-1/archer.txt [hunk 2]
```diff
@@ -117,10 +117,12 @@ payon_in02,64,71,4	script	Archer Guildsman#archer	85,{
 					close;
 				}
 			}
-			mes "[Archer Guildsman]";
-			mes "Well, alright.";
-			mes "See you next time.";
-			close;
+			else {
```
Comment (Atemo) [inline npc/pre-re/jobs/1-1/archer.txt:120]: With this "else", the above script continues (select case 1) and the player has access to dialog to which he didn't have access.

btw I don't see what was the problem here :
```
	mes "[Archer Guildsman]";
	mes "Alright, let me check.";
	next;
	mes "[Archer Guildsman]";
	mes "Well, alright.";
	mes "See you next time.";
	close;
```
seems "fine"

Comment (Daegaladh) [reply npc/pre-re/jobs/1-1/archer.txt:120]: According to the official script, after selecting "Yes, I am.", it should continue like this: 

```
mes "[Archer Guildsman]";
mes "Alright, let me check.";
next;

mes "[Archer Guildsman]";
mes "Are you..." + strcharinfo(0) + "?";
```

so yeah, the else is actually needed, and should only appear when you select "Cancel" in the Apply/Cancel menu

**Review by Daegaladh — APPROVED** [review]

**Pull request merged**